pkg/packagekit/updates.jsx

```diff
@@ -497,56 +742,185 @@ class ApplyUpdates extends React.Component {
+
+    if (reboot.length > 0) {
+        const col1 = reboot.slice(0, Math.floor(reboot.length / 2));
+        const col2 = reboot.slice(Math.floor(reboot.length / 2) + 1);
+        const expandedContent = (
+            <Flex>
+                <Flex flex={{ default: 'flex_1' }}>
+                    <FlexItem>
+                        <TextContent>
+                            <TextList>
+                                {col1.map(service => (<TextListItem key={service}>{service}</TextListItem>))}
+                            </TextList>
+                        </TextContent>
+                    </FlexItem>
+                </Flex>
+                <Flex flex={{ default: 'flex_1' }}>
+                    {col2.length > 0 && <FlexItem>
+                        <TextContent>
+                            <TextList>
+                                {col2.map(service => (<TextListItem key={service}>{service}</TextListItem>))}
+                            </TextList>
+                        </TextContent>
+                    </FlexItem>}
+                </Flex>
+            </Flex>
+        );
+
+        let title = cockpit.ngettext("$0 service need to be restarted", "$0 services need to be restarted", restart.length);
+        title = (<>
+            <ProcessAutomationIcon />
+            <span className="update-success-table-title">{cockpit.format(title, restart.length)}</span>
+        </>);
+
+        entries.push({
+            columns: [
+                { title },
+            ],
+            props: { key: "service" },
+            hasPadding: true,
+            expandedContent,
+        });
     }
-    stateStr = cockpit.format(stateStr, numUpdates, numSecurity);
+    if (restart.length > 0) {
+        const col1 = restart.slice(0, Math.floor(restart.length / 2));
+        const col2 = restart.slice(Math.floor(restart.length / 2) + 1);
+        const expandedContent = (
+            <Flex>
+                <Flex flex={{ default: 'flex_1' }}>
+                    <FlexItem>
+                        <TextContent>
+                            <TextList>
+                                {col1.map(service => (<TextListItem key={service}>{service}</TextListItem>))}
+                            </TextList>
+                        </TextContent>
+                    </FlexItem>
+                </Flex>
+                <Flex flex={{ default: 'flex_1' }}>
+                    {col2.length > 0 && <FlexItem>
+                        <TextContent>
+                            <TextList>
+                                {col2.map(service => (<TextListItem key={service}>{service}</TextListItem>))}
+                            </TextList>
+                        </TextContent>
+                    </FlexItem>}
+                </Flex>
+            </Flex>
+        );
 
-    if (!stateStr)
-        return null;
+        let title = cockpit.ngettext("$0 package need a system reboot", "$0 packages need a system reboot", reboot.length);
```

Comment:
I think it's nicer writen as const title and then the replace inline. 
You can break the content of the <span> in it's own line to avoid long lines.

But title is used as a constant bellow, let's keep it as such.